css/style.css

```diff
@@ -516,6 +516,49 @@ hr {
   .nav-ul a {
     display: none;
   }
+
+  .m-menu-continer {
+    display: none;
+    position: fixed;
+    top: 0;
+    left: 0;
+    height: 100vh;
+    width: 100%;
+    background-color: rgb(96, 112, 255, 0.9);
+    backdrop-filter: blur(3px);
+    overflow: hidden;
+  }

```

Comment:
[OPTIONAL] As mentioned by the previous reviewer, I also think it would be better if you remove the **scrolling**  when you are watching the **Mobile menu**, this will prevent users to interact with content in the sub-background.  You can use a CSS property `overflow: hidden`; to do that, which will make your mobile menu correspond to the original design and behaviors professional.